user/docker.md

```diff
@@ -0,0 +1,125 @@
+---
+title: Using Docker in Builds
+layout: en
+permalink: /user/docker/
+---
+
+<div id="toc"></div>
+
+Travis CI builds can run and build Docker images, and can also push images to
+Docker repositories or other remote storage.
+
+<!-- @joshk -->
+
+If you don't already have access to the beta workers running Ubuntu Trusty
```

Comment:
From what I've heard from internal communication this isn't true anymore, it'll be generally available.